nixos/modules/services/desktop-managers/cosmic.nix

```
@@ -0,0 +1,166 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.services.desktopManager.cosmic;
+in
+{
+  meta.maintainers = with lib.maintainers; [
+    thefossguy
+  ];
+
+  options = {
+    services.desktopManager.cosmic = {
+      enable = lib.mkEnableOption "COSMIC desktop environment";
```

```suggestion
      enable = lib.mkEnableOption "the COSMIC desktop environment";
```
`mkEnableOption` doesn't add the article.